packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -204,7 +204,7 @@ export function getConstantByNetwork<K extends ConstantKeys>(name: K, network: N
   }
 
   let value = constants[network as Network][name];
-  if (!value) {
+  if (value == null) {
```

Comment:
This is a fix for when the value is 0.

I discovered this bug when I was requesting `scheduledPaymentFeeFixedUSD` which is specified 0 in goerli, for example. The code threw an error but it should return 0.